_maps/templates/listeningbasics.dmm

```diff
@@ -0,0 +1,131 @@
+	dir = 1
+	},
+/obj/structure/window/reinforced/survival_pod{
+	dir = 4
+	},
+/obj/structure/window/reinforced/survival_pod{
+	dir = 5
+	},
+/obj/machinery/computer/records/medical/syndie,
+/turf/open/floor/iron/dark/small,
+/area/template_noop)
+"m" = (
+/obj/structure/window/reinforced/survival_pod{
+	dir = 1
+	},
+/obj/structure/window/reinforced/survival_pod{
+	dir = 8
+	},
+/obj/structure/window/reinforced/survival_pod{
+	dir = 9
+	},
+/obj/machinery/computer/message_monitor,
+/turf/open/floor/iron/dark/small,
+/area/template_noop)
+"t" = (
+/obj/machinery/door/window/survival_pod,
+/turf/open/floor/mineral/plastitanium/red,
+/area/template_noop)
+"y" = (
+/obj/effect/turf_decal/skyrat_decals/ds2/right,
+/obj/structure/window/reinforced/survival_pod{
+	dir = 4
+	},
+/obj/structure/table/reinforced,
+/obj/machinery/fax{
+	allow_exotic_faxes = 1;
+	syndicate_network = 1;
+	name = "Menacing Fax Machine";
+	color = "#FF0000";
+	fax_name = "Unknown Syndicate Fax - PLP"
+	},
+/turf/open/floor/mineral/plastitanium/red,
+/area/template_noop)
+"C" = (
+/obj/effect/turf_decal/skyrat_decals/ds2/middle,
+/obj/structure/chair/office/tactical{
+	dir = 1
+	},
+/turf/open/floor/mineral/plastitanium/red,
+/area/template_noop)
+"F" = (
+/obj/structure/window/reinforced/survival_pod,
+/obj/structure/window/reinforced/survival_pod{
+	dir = 4
+	},
+/obj/structure/window/reinforced/survival_pod{
+	dir = 10
+	},
+/obj/structure/window/reinforced/survival_pod{
+	dir = 6
+	},
+/obj/structure/filingcabinet/chestdrawer,
+/turf/open/floor/iron/dark/small,
+/area/template_noop)
+"Q" = (
+/obj/effect/turf_decal/skyrat_decals/ds2/left,
+/obj/structure/window/reinforced/survival_pod{
+	dir = 8
+	},
+/obj/structure/table/reinforced,
+/obj/item/paper_bin{
+	pixel_x = -5;
+	pixel_y = 6
+	},
+/obj/item/stamp/chameleon{
+	pixel_x = 7;
+	pixel_y = 12
+	},
+/obj/item/pen/fourcolor{
+	pixel_x = -5;
+	pixel_y = 7
+	},
+/obj/item/paper/crumpled/fluff{
+	pixel_x = 7;
+	default_raw_text = "<h1><table bgcolor="darkred" width="100%"><th><font color="#ffffff" size=4 face="OCRAStd,monospace"><div align="center">♦♦♦♦♦♦♦♦◹§◺♦♦♦♦♦♦♦♦</font></div></th></table></h1><hr /><font face="Veranda" size=3>Welcome to your very own "Portable" Listening Post! Now that you've unboxed the kit, lets get started on how everything works!<br><br>Included in this kit are:<br>* State of the art protective windows, from the dirty outside world!<br>* A computer bay fit to make the most hackerman of boys cry!<br>* A fax machine painted in sleek Syndicate Red.<br>* A filing cabinet! It's not special. Just a filing cabinet.<br><br>Listen to your foes! Stink up their comms! Find a cute girl to stalk! All is possible with the "Portable" Listening Post!<br><hr />The fax machine included in the PLP is set up to the Sothran network. It is not visible on any fax machines aboard the station, while still able to send to them. Use this to your advantage.</font><hr><center><font size=3>❘❙❚❘❙❚|<span style="color:black;
```

Comment:
I'm pretty sure this is missing airquotes /somewhere/ in this mess

Reply:
Everywhere. Basically just wrap the whole thing with {} and it should be good.

Reply:
Example: https://github.com/Bubberstation/Bubberstation/blob/master/modular_zubbers/code/game/machinery/burger_reactor/reactor_guide.dm